ReactiveObjC/RACDelegateProxy.m

```diff
@@ -46,7 +46,9 @@ - (BOOL)isProxy {
 }
 
 - (void)forwardInvocation:(NSInvocation *)invocation {
-  [invocation invokeWithTarget:self.rac_proxiedDelegate];
+  if (self.rac_proxiedDelegate) {
```

Comment:
Since `self. rac_proxiedDelegate` is `weak`, let's store it in a `strong` variable first to ensure it doesn't get nullified in the middle.